### PR TITLE
moving to 1.0.0, combining user info and attributes in calls

### PR DESCRIPTION
**Pull request by usmanity**
- adding timestamp
- updating readme

### CODE REVIEW

File: lib/outbound.rb [hunk 1]
```diff
@@ -34,23 +34,22 @@ def Outbound.init api_key, log_level=Logger::ERROR
...
       @logger.error res.error
       return res
     end
-
-    return @ob.identify user_id, info, attributes
+    return @ob.identify user_id, info
   end
 
-  def Outbound.track user_id, event, properties={}, user_info={}, user_attributes={}
+  def Outbound.track user_id, event, properties={}, user_info={}
```
Comment (MrTravisB) [inline lib/outbound.rb]: don't you need to add a `timestamp` param to this method as well?

File: lib/outbound.rb [hunk 2]
```diff
@@ -34,23 +34,22 @@ def Outbound.init api_key, log_level=Logger::ERROR
...
     if @ob == nil
       res = Result.new Outbound::ERROR_INIT, false
       @logger.error res.error
       return res
     end
-    return @ob.track user_id, event, properties, user_info, user_attributes
+    return @ob.track user_id, event, properties, user_info
```
Comment (MrTravisB) [inline lib/outbound.rb]: then pass it through here

**Comment by MrTravisB** [comment]
Couple comments. Make sure timestamp works. With what you've done, a timestamp will always be set, it just doesn't accept one as a parameter so the library makes the timestamp every time.

File: lib/outbound.rb (hunk 2 above)
Comment (usmanity) [inline lib/outbound.rb]: ✔︎

File: lib/outbound.rb (hunk 1 above)
Comment (usmanity) [inline lib/outbound.rb]: ✔︎

**Pull request merged**